config/connect-streams-file-source.properties

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name=local-file-source
```

Comment:
Not sure I understand the question here -- I just meant that the configs look basically the same and it seems unnecessary to duplicate. If we just changed the topic to, e.g., "file-input", streams could still use it without duplicating everything. The "streams" prefix isn't really important here and in fact emphasizing that the data can come from Connect and isn't specific to Streams seems like what you want.